src/queries/useGetRandomMeme.js

```diff
@@ -0,0 +1,22 @@
+import { useQuery } from "react-query";
+import request from "hooks/request/request";
+import { QUERY_KEYS } from "utils/constants";
+
+const { MEME } = QUERY_KEYS;
+
+const getRandomMeme = () => {
+  const response = request();
+
+  return response;
+};
+
+export default (isButtonPressed) => {
```

Comment:
Probably... there is a way to reset cache when trigger this hook.
Because this is just for getting random meme which doesn't need to catch any data.

So far, because of cache, display prior data image when press meme button to get another meme image.

Reply:
Ok found it.
clear cache by useClientQuery hook from react-router.
Use it in memes.jsx when press reset button after getting meme.

So... now prior image won't show up even re-press MEME button.

https://user-images.githubusercontent.com/44974307/147408791-22bcc912-689c-43c7-a4a3-7b37d4867887.mov